src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutRecoverabilityBehavior.cs

```diff
@@ -7,6 +7,7 @@ namespace NServiceBus
     using System;
     using System.Collections.Generic;
     using System.Threading.Tasks;
+    using JetBrains.Annotations;
```

Comment:
@andreasohlund @Particular/nservicebus-maintainers To be honest, I think it's a valid question to ask. To make the discussion going, here is my thinking when deciding to use it:

- We have everything set up to use it,
- We all use R# so it might benefit us when writing code,
- We use code inspections on TeamCity,
- We were using these attributes already in some places.
- It does not provide any binary dependency on JetBrains stuff.
- We already do have comments in the code that silence R# warnings, as far as I can tell.

I also see no issue in combining multiple static analysis tools: R#, Roslyn analyzers, etc.

I guess the real question here is: do we want to litter our source code with hints to static analysis tools? I guess this also applies to things like `#pragma warning disable`.